graphql/schema/types/distro.graphql

```diff
@@ -42,6 +42,31 @@ enum DispatcherVersion {
   REVISED_WITH_DEPENDENCIES
 }
 
+enum Arch {
+  LINUX_32_BIT
+  LINUX_64_BIT
+  LINUX_ARM_64_BIT
+  LINUX_PPC_64_BIT
+  LINUX_ZSERIES
+  OSX_64_BIT
+  OSX_ARM_64_BIT
+  WINDOWS_32_BIT
+  WINDOWS_64_BIT
+}
+
+enum CommunicationMethod {
+  LEGACY_SSH
+  SSH
+  RPC
+}
+
+enum BootstrapMethod {
+  LEGACY_SSH
+  SSH
+  USER_DATA
+}
+
+
```

Comment:
nit: did you intend to put 3 newlines in between this + the next section?

Reply:
fixed!